example/image-classification/symbol/resnext.py

```diff
@@ -0,0 +1,180 @@
+'''
+Adapted from https://github.com/tornadomeet/ResNet/blob/master/symbol_resnet.py
+Original author Wei Wu
+
+Implemented the following paper:
+
+Kaiming He, Xiangyu Zhang, Shaoqing Ren, Jian Sun. "Identity Mappings in Deep Residual Networks"
```

Comment:
the paper is resnext: Aggregated Residual Transformations for Deep Neural Networks